### ISSUE TITLE
When Number zapping - video background is reduced in size

### ISSUE THREAD

**Issue by hmiah101** [OpenViX/skins#9]
Hi,

There seems to be an issue when you type in numbers to change channel. For example, if I type "101" on my remote control, "101 BBC One HD" is shown on the centre of the screen - that is perfectly fine. However, this also makes the video playing in the background reduce in size to a smaller box on the top left of the screen with the rest of the background in black. I'm pretty sure that's not intended to happen for number zapping and only meant to be like that when browsing the menus and the EPG as it currently is.

Would really appreciate a fix for this problem. Thanks!
![image](https://cloud.githubusercontent.com/assets/10728400/12238379/a33c1504-b87b-11e5-8262-0b178ec2c249.JPG)

**Comment by RobvanderDoes** [comment]
1- Which skin are you using?
2- Is this during live-TV?

**Comment by hmiah101** [comment]
Thanks for getting back. I'm using Magic HD Night. I've also noticed this happens on the Magic HD Noire skin which I believe shares a common skin xml. It does not happen with the original Magic HD skin. 

As regards to whether this is during Live TV, yes this occurs when watching live TV. I have not tried this with IPTV or streaming local video content. Hope that helps and look forward to a resolution.

**Comment by delagroov** [comment]
Hello

I can not reproduce this issue with the Magic hd night/noire skin

**Comment by hmiah101** [comment]
Hi,

Apologies for the late reply. I do not have any resolution-specific settings set on the box, nor are there any tv resolution settings which have been set. I've flashed my box around 4-5 times with a fresh start over the last 5 months and the issue still seems to persist. Not really sure where the root issue lies.... :/

**Comment by RobvanderDoes** [comment]
On which STB does this happen?

**Comment by hmiah101** [comment]
Hi,

It's the VU+ Solo 2

**Comment by RobvanderDoes** [comment]
I tried to reproduce your issue (Solo2/Magic HD Night/ViX 4.1.011), but I can't.
I can only suggest it's due to a corrupted image or a corrupting plugin.
Which image do you use?

**Comment by hmiah101** [comment]
I'm currently using OpenVix 4.1.011, though this also occurred back when I used to use OpenATV 5.2 and previous Vix versions. If it's not occurring for you and no one else has reported it, I guess it's something on my end that's causing this problem as you suggested. The only problem is, I have no idea what's causing this and I've flicked through the settings over the last few months to find out what's causing it but have had no luck. If anyone has any suggestions, I'd be very grateful. ;)

Many thanks for your support and time RobvanderDoes.
